bin/GameIndex.dbf

```diff
@@ -7672,18 +7671,22 @@ Compat = 5
 Serial = SLES-50506
 Name   = Half-Life
 Region = PAL-Unk
+Compat = 5
 ---------------------------------------------
 Serial = SLES-50507
 Name   = Half-Life
 Region = PAL-F
+Compat = 5
 ---------------------------------------------
 Serial = SLES-50508
 Name   = Half-Life
 Region = PAL-I
+Compat = 5
 ---------------------------------------------
 Serial = SLES-50509
 Name   = Half-Life
 Region = PAL-S
+Compat = 5
 ---------------------------------------------
```

Comment:
If you didn't test any of these regions, please revert whatever compatibility change was made.

Reply:
It also needs a full report in the compatibility list on the forum and an updated compatibility rating on the website as well. And then only update it in the index for the region that was tested, not for any other version.